src/components/admin/AdminContentList.tsx

```diff
@@ -38,6 +39,14 @@ const AdminContentList: React.FC<Props> = props => {
   const [searchText, setSearchText] = useState("");
   const [isJudging, setIsJudging] = useState<any>(undefined);
 
+  const [isOpen, setIsOpen] = useState(true);
```

Comment:
all of these can be removed since we are now just always showing the description.